lib/avrolution/configuration.rb

```diff
@@ -0,0 +1,42 @@
+module Avrolution
+
+  COMPATIBILITY_SCHEMA_REGISTRY_URL = 'COMPATIBILITY_SCHEMA_REGISTRY_URL'.freeze
+
+  class << self
+    # Root directory to search for schemas, and default location for
+    # compatibility breaks file
+    attr_writer :root
+
+    # Path to the compatibility breaks file. Defaults to
+    # #{Avrolution.root}/avro_compatibility_breaks.txt
+    attr_writer :compatibility_breaks_file
+
+    # The URL (including any Basic Auth) for the schema registry to use for
+    # compatibility checks
+    attr_writer :compatibility_schema_registry_url
+
+    attr_accessor :logger
+  end
+
+  @logger = Avrolution::PassthruLogger.new($stdout)
+
+  def self.root
+    raise('root must be set') unless @root
```

Comment:
I thought we decided in another PR recently that it was better to lead with the `raise` because that was more intention revealing than making it the last condition!